recipe/meta.yaml

```diff
@@ -42,21 +43,23 @@ outputs:
   - name: implicit
     build:
       number: {{ number }}
-      script: python -m pip install . --no-deps -vv
+      script_env:
+        - CMAKE_GENERATOR = "Ninja"
+      script: python -m pip install --no-index --no-deps --ignore-installed --cache-dir pip_cache . -vv
 
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler not in (undefined, "None")]
-        - make  # [linux]
+        - cmake
+        - ninja
       host:
         - python
         - pip
+        - setuptools >=42
```

Comment:
Despite having `setuptools` constrained to match [this upstream `setuptools` constraint]( https://github.com/benfred/implicit/blob/91bd56c7fccd9d72bbde69bd958713605ba8bcf5/pyproject.toml#L7 ), am still [seeing failures on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=735051&view=logs&j=e9afaa34-1a0f-534e-78fc-fae528ccd915&t=48395e60-bc5f-5e45-7455-d404f0c95f3a&l=263 ):

```
  ERROR: Could not find a version that satisfies the requirement setuptools>=42 (from versions: none)
  ERROR: No matching distribution found for setuptools>=42
```